robotick/knitware-shared/cpp/src/workloads/brain/BreathingWorkload.cpp

```
@@ -0,0 +1,126 @@
+			{
+				const float u = (exhale > 1e-6f) ? (p / exhale) : 1.0f;
+				return lerp(1.0f, -1.0f, smoothstep01(u)); // +1 -> -1
+			}
+
+			return -1.0f;
+		}
+
+		void setup()
+		{
+			outputs = {};
+			outputs.breathing_wave = -1.0f;
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)